src/components/DataSection/DataSection.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import './DataSection.scss';
+import { getLocation } from '../../commons/helpers';
+import Card from '../Card/Card';
+import { Tabs } from '../../statics/Statics';
+
+function DataSection ({activeTab}) {
+    const [data, setData] = useState([]);
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => getData(), [activeTab])
+
+    const callback = (results, status) => {
+     if (status === window.google.maps.places.PlacesServiceStatus.OK) {
+            setData(results)
+            setLoading(false)
+        }
+    }
+    const getData = () => {
+        setLoading(true)
+        getLocation().then(({ coords: {latitude, longitude} }) => {
+            var center = new window.google.maps.LatLng(latitude,longitude);
+            var map = new window.google.maps.Map(document.getElementById('map'), {
+                center,
+                zoom: 15
+            });
+            var request = {
+                location: {lat: latitude, lng: longitude},
+                radius: '500',
+                type: [activeTab]
+            };
+            let service = new window.google.maps.places.PlacesService(map);
+            service.nearbySearch(request, callback);
```

Comment:
There's no need to create a new service every time before fetching the data. Only send the request with the changes.